tests/switchless_threads/host/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+add_custom_command(
+  OUTPUT switchless_threads_u.h switchless_threads_u.c switchless_threads_args.h
+  DEPENDS
+  edger8r
+  ../switchless_threads.edl
+  COMMAND edger8r --experimental --untrusted --search-path ${CMAKE_CURRENT_SOURCE_DIR}/.. switchless_threads.edl)
```

Comment:
Is --experimental still needed here?

Reply:
We haven't decided whether to drop it altogether or use a `--preview` flag. @radhikaj

Reply:
We could rename experimental to preview, I'm fine with that. (In fact, I quite like it.)

Reply:
Regardless of the name (I have no preference between preview vs experimental), the edger8r support is a preview and the rest of the switchless feature has a hard dependency on it, then the whole switchless feature should be a preview in my view.

Reply:
I think we can go with --preview for switchless support. 
@dthaler : Do we have documentation that edger8r is in preview right now?

Reply:
We can switch --experimental to --preview in a different PR.

Reply:
Opened Issue #2203 to track this.